### DISCUSSION TITLE
Full/empty storage containers feature

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#6593] [Ideas and suggestions]
This feature would affect toolbelts, toolboxes, the new bandoliers and heavy scooters, and basically every mobile item container smaller than a crate.

Simple: When an item container such as the toolbelt has any amount of items inside it, it cannot be placed inside another mobile or wearable container (such as crates and the bandolier). But when the container is empty, you're allowed to place it inside another container.

This would make looting a bit more entertaining and instantly end a lot of non-intended metas that just screw all over another features.

**Comment by HeatedWafflez** [comment]
![flopper](https://user-images.githubusercontent.com/51425624/132154912-07481118-1541-47f8-8070-896fa5282b00.gif)

**Reply by LePiromano** [reply]
i disagree

**Comment by MateusJK15** [comment]
It wouldn't make looting more entertaining, just slower and boring. Explore a wreck/ruin, kill the monsters, that is entertaining, but transport items from one place to another ? The less time it takes, the better, there's no fun in it.

**Comment by V1Htor** [comment]
Just add a "backpack" that can store only ores

**Comment by ubertpendragon** [comment]
Needed like yesterday

Recursive inventory is one of the biggest balance problems. You want to be able to store containers, but as long as those containers can contain items inside items, total storage can be multiplied several times

Right now you can put toolbelts into crates, resulting in 120 inventory slots: 
![image](https://user-images.githubusercontent.com/73229309/132254515-15177e56-de64-4ac8-a256-1df0a4b9a249.png), meaning near-infinite inventory space for looting
